chat-plugins/othermetas.js

```diff
@@ -147,6 +147,109 @@ exports.commands = {
 	},
 	mixandmegahelp: ["/mnm <pokemon> @ <mega stone> - Shows the Mix and Mega evolved Pokemon's type and stats."],
 
+	'!stone': true,
+	move: 'stone',
```

Comment:
This should not be an alias, /move already exists

Reply:
Oh yeah it does what are you doing overriding core sim features???

Reply:
oOPS